### PR TITLE
First API route complete for GET project by id

### PR DESCRIPTION
**Pull request by zhujohnny**

### CODE REVIEW

File: client/src/components/Stats.js [hunk 1]
```diff
@@ -66,13 +66,13 @@ class Stats extends React.Component {
 
         <div id="pledge" style={style.after}>
           <div style={style.pledged}>
-            ${this.props.data.product.amtPledged}
+            ${this.props.data.total_pledge_amt}
```
Comment (CareyL96) [inline client/src/components/Stats.js]: no snake case

Comment (careyl96) [inline client/src/components/Stats.js]: no snake case

File: client/src/components/Stats.js [hunk 2]
```diff
@@ -66,13 +66,13 @@ class Stats extends React.Component {
...
           </div>
-          <div>pledged of {this.props.data.target.amt} goal</div>
+          <div>pledged of {this.props.data.goal_amt} goal</div>
```
Comment (careyl96) [inline client/src/components/Stats.js]: no snake case

File: client/src/components/Stats.js [hunk 3]
```diff
@@ -66,13 +66,13 @@ class Stats extends React.Component {
...
         </div>
 
         <div id="backers" style={style.after}>
-          <div style={style.nums}>{this.props.data.product.numBackers}</div>
+          <div style={style.nums}>{this.props.data.pledges_count}</div>
```
Comment (careyl96) [inline client/src/components/Stats.js]: no snake case

**Review by careyl96 — CHANGES_REQUESTED** [review]
change variables to non-snake case

File: client/src/components/Stats.js (hunk 1 above)
Comment (zhujohnny) [reply client/src/components/Stats.js]: Changed to camelCase

File: client/src/components/Stats.js (hunk 2 above)
Comment (zhujohnny) [reply client/src/components/Stats.js]: Changed to camelCase

File: client/src/components/Stats.js (hunk 3 above)
Comment (zhujohnny) [reply client/src/components/Stats.js]: Changed to camelCase

**Pull request closed without merging**